ansible/vars/aeternity/integration.yml

```diff
@@ -1,14 +1,14 @@
-datadog_enabled: yes
 api_base_uri: http://{{ public_ipv4 }}:{{ node_config.http.external.port }}/v2
-genesis_accounts:
-  "ak_UAzhn9rAQg568v6Hwt3w2HPaQb9X9Nw6JbLmnv7trhmGmWGGp": 100000000001
 configure_peers: false
 seed_peers:
   - "35.177.40.45"
```

Comment:
this is not related to the node configuration, but to static IP configuration of the instances